server/src/main/java/org/apache/iotdb/db/conf/directories/strategy/DirectoryStrategy.java

```diff
@@ -33,19 +35,17 @@
   protected static final Logger logger = LoggerFactory.getLogger(DirectoryStrategy.class);
 
   /**
-   * All the folders of data files, should be init once the subclass is created.
+   * All the folders of data files, should be setFolders once the subclass is created.
    */
-  List<String> folders;
+  List<String> folders = new ArrayList<>();
```

Comment:
In normal situations, this new ArrayList is useless, is this initialization necessary?

Reply:
Just in case of `NullPointer`. which may be unnecessary. I have removed it.